### PR TITLE
new version, working with the patched phantom release, the submit has…

### PR DESCRIPTION
**Pull request by govoni**
… also a test option to produce events locally

### CODE REVIEW

**Comment by efeyazgan** [comment]
Please remove the unnecessary commented lines.

**Comment by govoni** [comment]
which ones?

File: bin/Phantom/cards/examples/phantomProduction_example.cfg [hunk 1]
```diff
@@ -14,13 +14,15 @@
 # CMSSW environment for the gridpack creation
 # this is NOT necessarily the one used in the detector simulation and reconstruction,
 # it just sets the environment for the phantom running
-CMSSW = CMSSW_9_2_11
+# CMSSW = CMSSW_9_2_11
```
Comment (efeyazgan) [inline bin/Phantom/cards/examples/phantomProduction_example.cfg]: Commented inputs like this

**Comment by govoni** [comment]
these are not unnecessary: associating scram_arch and cmssw release is always unnecessarily painful for cms, as that info in hidden. In this way, in the example we have an example for both a 2016 and 2017 configuration.

**Pull request merged**